cttportfolio.css

```diff
@@ -3,168 +3,486 @@
 }
 html{
 	font-size:62.5%;
+	font-family:'Montserrat', sans-serif;
+	color:rgba(4,8,9,1);
 }
 body{
 	margin:0;
+	background-color:rgb(2,8,9);
 }
 main > div{
 	margin:0 auto;
-	max-width:1350px;
-	padding:0 1%;
+	max-width:1300px;
+}
+h4, h5, h6,.pp_body h1{
+	font-weight:700;
+	font-size:6.0rem;
+	text-transform:uppercase;
+	-webkit-text-stroke-width: 0.3px;
+	-webkit-text-stroke-color: black;
+	color: rgb(248,248,255);
+}
+h5{
+	font-weight:600;
+	font-size:4rem;
+}
+.am_text, button, .navlinks a, .contactmetext > div, img{
+	box-shadow: 0px -2px 5px rgba(2,8,9,0.6)
+}
+button{
+	color:white;
+	background:none;
+	padding:10px;
+	font-size:2rem;
+	text-transform:uppercase;
+	border:1px solid rgb(248,248,255);
+	font-weight:600;
+	border-radius:10px;
+}
+a.icon_hover img, a.pf_sub_links h6, .navlinks.icon_hover > a:hover{
+	border:1px solid rgba(248,248,255,0);	
+}
+a.icon_hover:hover img, a.pf_sub_links:hover h6, button:hover,.navlinks.icon_hover > a:hover{
+	border:1px solid rgba(248,248,255,1);
+	box-shadow: 0px -2px 5px rgba(248,248,255, 0.6);
+}
+.background_color,.project_text > * {
+	background-color:rgb(255,182,193,0.6);
```

Comment:
`rgb` only accepts three arguments, you're supplying it with four. Did you mean `rgba()`?